src/WpfMath/TexPredefinedFormulaParser.cs

```diff
@@ -194,9 +197,13 @@ public TexFormula Result
             public override void Parse(SourceSpan source, XElement element)
             {
                 var name = element.AttributeValue("name");
-                var result = this.TempFormulas[name];
-                Debug.Assert(result != null);
-                this.Result = result;
+                if (this.TempFormulas.ContainsKey(name))
```

Comment:
I don't think we should have this check here either. If the user have defined a `<Return>` block referencing a nonexisting `<Formula>`, then it should be an error. Such cases shouldn't be silently ignored.